package.json

```diff
@@ -8,11 +8,14 @@
   "main": "index.js",
   "scripts": {
     "start": "./bin/start",
+    "start-debug": "node --inspect ./import.js",
+    "import": "node --max_old_space_size=8192 ./import.js",
```

Comment:
This `import` task just duplicates what is in `./bin/start`. Please remove it and if you'd like to add it, do so in a separate PR.

I think some other importers do define an `import` script so it could be interesting to insure consistency, although in general I think `npm start` is enough.